libraries/helpers.rb

```diff
@@ -19,25 +19,35 @@
 
 module StrongDM
   module Helpers
-    def sdm_relay_token(type = 'relay')
+    def sdm_relay_token(admin_token, name, type = 'relay', ip = nil, port = 5000, bind_ip = '0.0.0.0', bind_port = 5000)
```

Comment:
You probably don't need these default values, since you have the same defaults in the thing that calls this function.

Reply:
As a library, it ends up being available to anyone to use, so I'd rather keep it a clean interface, even if I'm doing the right thing when I consume it within my resource.